ngrinder-runtime/src/main/java/org/ngrinder/sm/NGrinderSecurityManager.java

```diff
@@ -50,9 +56,25 @@
 	}
 
 	void init() {
+		controllerHomeDir = resolveControllerHomeDir();
```

Comment:
Why does it resolve controller home? is it better to name it with HomeDir?  It's because this class not only called from controller but agent.

Reply:
However, this code can be executed agent machine.. and mistakenly configured env var can cause the error.
I think it might be better to provide the way to detect ngrinder controller context.
Can we inherit PropertyBuilder (named ValidationPropertyBuilder) and add the additional param in buildJVMArgumentWithoutMemory method and initiate it in the LocalScriptTestDriveService ?

Reply:
Yes, I understand.

I will change the read access logic in security manager according to the jvm argument "ngrinder.context".

Reply:
Thx

Reply:
Still is it better to resolve ngrinder home only in controller context?